Butane-Schema.json

```diff
@@ -6,63 +6,124 @@
   "type": "object",
   "required": ["variant", "version"],
 
-  "properties": {
+"properties": {
     "variant": {
       "$id": "#/properties/variant",
       "type": "string",
       "title": "variant (string):",
-      "description": "Used to differentiate configs for different operating systems. Must be fcos for this specification.",
-      "enum": ["fcos"],
+      "description": "Used to differentiate configs for different operating systems.",
+      "enum": [
+         "fcos",
+         "flatcar"
+      ],
       "default": "fcos"
     },
     "version": {
       "$id": "#/properties/version",
       "type": "string",
-      "title": "version (string):",
-      "enum": ["1.4.0", "1.5.0", "1.6.0"],
-      "default": "1.6.0"
+      "title": "version (string):"
     }
   },
-  "oneOf": [
+  "allOf": [
     {
-      "allOf": [
+      "oneOf": [
         {
           "properties": {
+            "variant": {
+              "const": "fcos"
+            },
             "version": {
-              "const": "1.4.0"
+              "enum": [
+                "1.4.0",
+                "1.5.0",
+                "1.6.0"
+              ],
+              "default": "1.6.0"
             }
           }
         },
-        {
-          "$ref": "https://raw.githubusercontent.com/Relativ-IT/Butane-Schemas/Release/v1.4.0/butane-v1.4.0.json"
-        }
-      ]
-    },
-    {
-      "allOf": [
         {
           "properties": {
+            "variant": {
+              "const": "flatcar"
+            },
             "version": {
-              "const": "1.5.0"
+              "enum": ["1.1.0"],
+              "default": "1.1.0"
             }
           }
-        },
-        {
-          "$ref": "https://raw.githubusercontent.com/Relativ-IT/Butane-Schemas/Release/v1.5.0/butane-v1.5.0.json"
         }
       ]
     },
     {
-      "allOf": [
+      "oneOf": [
         {
-          "properties": {
-            "version": {
-              "const": "1.6.0"
+          "allOf": [
+            {
+              "properties": {
+                "variant": {
+                  "const": "fcos"
+                },
+                "version": {
+                  "const": "1.4.0"
+                }
+              }
+            },
+            {
+              "$ref": "./v1.4.0/butane-v1.4.0.json"
```

Comment:
you should leave full path to Release branche : https://raw.githubusercontent.com/Relativ-IT/Butane-Schemas/Release/v1.4.0/butane-v1.4.0.json
This file is intended to be the one that users rely on ;)